ADRs/0003-llm-benchmark-configuration.md

```diff
@@ -0,0 +1,25 @@
+# LLM-0003 LLM model configuration
```

Comment:
oh this is conflicting with the other PR which isn't merged yet (depending on what is merged first)

Reply:
Why is it conflicting?

Reply:
The number